webapp/www/templates/client_setup.html

```diff
@@ -26,20 +26,22 @@ <h2>Thank you for participating - only two steps left!</h2>
                   </g>
               </g>
           </svg>
-        <h4>Install the {{base_app}} app</h4>
+        <h4>Install the e-mission app</h4>
```

Comment:
aren't you going to have people install the emTripLog app, not the e-mission app?